.NET/Microsoft.Recognizers.Text.DateTime/DateTimeRecognizer.cs

```diff
@@ -97,12 +98,21 @@ protected override void InitializeConfiguration()
                         new GermanMergedParserConfiguration(new BaseOptionsConfiguration(options))),
                     new BaseMergedDateTimeExtractor(new GermanMergedExtractorConfiguration(options))));
 
+            // TODO to be uncommented when all tests for Dutch are green.
+            RegisterModel<DateTimeModel>(

```

Comment:
This should be kept commented out before we can merge the PR.

Reply:
#resolved